requirements.txt

```diff
@@ -0,0 +1,3 @@
+pygame==2.0.1
```

Comment:
i don't want to include pygame/pyglet becuase they are optional

Reply:
Makes sense. There was a line in the tests that directly import util_pygame. Patching that up now...